src/components/RaceCards/RaceCard.tsx

```diff
@@ -0,0 +1,113 @@
+import Link from 'next/link';
+import { motion } from 'framer-motion';
+import styles from './RaceCard.module.scss';
+
+type RaceCardProps = {
+    header: string;
+    url?: string
+    abilityBonuses: Array<any>;
+    traits: Array<any>;
+};
+
+function RaceCard({
+  url = 'races', traits, abilityBonuses, ...props
```

Comment:
you only need to type your props where you set defaults as your spreading them

```suggestion
  url = 'races', ...props
```